specification/web/resource-manager/Microsoft.Web/stable/2019-08-01/CommonDefinitions.json

```diff
@@ -42,7 +42,6 @@
         "name",
         "location",
         "virtualNetwork",
```

Comment:
Please remove this extra comma since the following line is deleted. The CI failures are possibly all caused by this.

Reply:
done